### PR TITLE
feat: NO-JIRA add types to themes

### PR DESCRIPTION
**Pull request by juliodialpad**
# Add types to themes

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [x] Feature

## :book: Description

- Added types to /themes/* entries.

## :bulb: Context

Added `themes/*` entries recently but haven't added types so TS was missing types for them.

(Tested on a local TS Vue 3 project)

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.

## :crystal_ball: Next Steps

- Release and test.

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { defineConfig } from 'vite';
 import path, { resolve } from 'path';
 import { fileURLToPath } from 'url';
+import dts from 'vite-plugin-dts';
```
Comment (braddialpad) [inline vite.config.js:4]: We didn't have to use this plugin for any of the other packages?

**Review by braddialpad — APPROVED** [review]
thanks

File: vite.config.js (hunk 1 above)
Comment (juliodialpad) [reply vite.config.js:4]: On the other packages we’re using `vue-tsc`, themes are JS only, that’s why we’re using `vite-plugin-dts` here

Comment (braddialpad) [reply vite.config.js:4]: Ah yeah, gotcha

**Comment by braddialpad** [comment]
Merging so alex can test.

**Pull request merged**